### ISSUE TITLE
Font stack is inconsistent with Barlesque

### ISSUE THREAD

**Issue by howieweiner** [bbc/gel-typography#47]
If $gel-type-enable--font-family is enabled, the font stack produced by gel-typography is inconsistent with that in Barlesque/Orb

Compare:
https://github.com/bbc/gel-typography/blob/master/_typography.scss#L41

With:
https://github.com/bbc/nav-orbit/blob/master/static-sources/sass/global/_reset.scss#L251
and
https://github.com/bbc/nav-barlesque/blob/a44453ec49a5edb99390b523f4a354a500757103/static-sources/orb/sass/global/_reset.scss#L251

As most teams will not need to enable the font-family flag, the demo pages for this project are misleading, as they appear to have been built with this flag enabled

**Comment by shaunbent** [comment]
@itsaljones can you add some thoughts to this?!?

**Comment by itsaljones** [comment]
Yeah it would be good to make them consistent and in-line with the Guidelines so: Helvetica, Ariel, sans-serif. 

I propose we tweak all of the above links to match. Thoughts? @howieweiner @shaunbent
